boto3/dynamodb/transform.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, *args, **kwargs):
             new_example_value='Key(\'mykey\').eq(\'myvalue\')'
         )
 
-        cond_expression_shape_docs = DocumentModifiedShape(
+        con_expression_shape_docs = DocumentModifiedShape(
```

Comment:
An expression a few lines down uses it and was a single character over the max. This was the simplest way to fix the issue.